docs/sources/developers/http_api/reporting.md

```diff
@@ -15,15 +15,446 @@ title: Reporting API
 
 This API allows you to interact programmatically with the [Reporting]({{< relref "../../dashboards/create-reports/" >}}) feature.
 
+> The Reporting API is not stabilized yet, it is still in active development and may change without prior notice.
+
 > Reporting is only available in Grafana Enterprise. Read more about [Grafana Enterprise]({{< relref "../../enterprise/" >}}).
 
 > If you are running Grafana Enterprise, for some endpoints you'll need to have specific permissions. Refer to [Role-based access control permissions]({{< relref "../../administration/roles-and-permissions/access-control/custom-role-actions-scopes/" >}}) for more information.
 
-## Send a report
+## List all reports
+
+`GET /api/reports`
+
+#### Required permissions
+
+See note in the [introduction]({{< ref "#reporting-api" >}}) for an explanation.
+
+| Action       | Scope                       |
+| ------------ | --------------------------- |
+| reports:read | reports:\*<br>reports:id:\* |
+
+### Example request
+
+```http
+GET /api/reports HTTP/1.1
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer eyJrIjoiT0tTcG1pUlY2RnVKZTFVaDFsNFZXdE9ZWmNrMkZYbk
+```
 
-> Only available in Grafana Enterprise v7.0+.
+### Example response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: 1840
+
+[
+	{
+		"id": 2,
+		"userId": 1,
+		"orgId": 1,
+		"dashboardId": 0,
+		"dashboardName": "Alerting with TestData",
+		"dashboardUid": "",
```

Comment:
Maybe we shouldn't include deprecated properties here?

Reply:
I agree.